### PR TITLE
Update uninstall driver test's configuration

### PR DESCRIPTION
**Pull request by leidwang**
1.Fix the bug of incorrect au3 script name
2.Copy the msi au3 script and msi file to same path to
aviod missing file issue
3.add script to run the test case under 32bit OS

ID:2096,2107

### CODE REVIEW

**Comment by leidwang** [comment]
Hi @xiagao @heywji 

Would you please help review this PR?Thanks.

I ran the test with this PR,but there still has some errors, but I think the error is not related to this PR.I will add the log file to the issue, would you please help to check if it is known issue？

**Comment by xiagao** [comment]
@leidwang From your log, the guest shut down, so it failed to get the installer info. And the os shutdown is due to the installer behavior changed. There is a patch https://github.com/autotest/tp-qemu/pull/3982 to fix it.

**Comment by leidwang** [comment]
> @leidwang From your log, the guest shut down, so it failed to get the installer info. And the os shutdown is due to the installer behavior changed. There is a patch #3982 to fix it.

Thank you @xiagao 
If there is no other concern about this PR, could you please ack this PR?Thanks!

**Comment by xiagao** [comment]
The code looks good to me. ACK.

**Review by heywji — APPROVED** [review]
LGTM

**Comment by leidwang** [comment]
Hi @vivianQizhu 
Would you please help review this PR?Thanks.

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg [hunk 1]
```diff
@@ -31,6 +34,9 @@
     i386:
         devcon_dirname += "x86"
         qemu_ga_pkg = qemu-ga-i386.msi
+        msi_name = "virtio-win-gt-x86.msi"
+        uninstall_msi_script = "uninstall_via_msi_x86.au3"
+        uninstall_msi_script_path = "WIN_UTILS:\${uninstall_msi_script}"
     x86_64:
         devcon_dirname += "amd64"
         qemu_ga_pkg = qemu-ga-x86_64.msi
```
[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg]: Okay, updated the code.Thanks!

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**